shared/js/background/events.es6.js

```diff
@@ -499,6 +499,44 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
 
         res(grade)
         return true
+    } else if (req.toggleSiteProtections) {
+        tabManager.setList({
+            list: 'allowlistOptIn',
+            domain: req.domain,
+            value: req.status
+        })
+        tabManager.setList({
+            list: 'allowlisted',
+            domain: req.domain,
+            value: req.status
+        })
+
+        return true
+    } else if (req.openFeedbackPage) {
+        const feedbackPath = '/html/feedback.html'
+        const url = chrome.runtime.getURL(feedbackPath)
+        chrome.tabs.create({ url: url })
+
+        return true
+    } else if (req.openBreakageReport) {
+        chrome.tabs.create({ url: req.url })
+
+        return true
+    } else if (req.breakageFlow) {
+        chrome.tabs.reload(req.tabId, { bypassCache: true }, () => {
+            setTimeout(() => {
```

Comment:
We should probably wait for the page navigation to complete right?

We should store the modal event in a global/storage and trigger it on _chrome.webNavigation.onDOMContentLoaded_ or similar.

So perhaps:
```
    } else if (req.breakageFlow) {
        browserWrapper.syncToStorage({ breakageModalLoad: req.tabId })
        chrome.tabs.reload(req.tabId, { bypassCache: true })
```

And then:
```
chrome.webNavigation.onDOMContentLoaded.addListener((load) => {
    browserWrapper.getFromStorage('breakageModalLoad', (modalId) => {
      if (moadId && load.tabId === moadId) {
        // Inject modal
      }
    })
}, {...})
```